packages/rich-text/src/RichTextEditor.styles.ts

```diff
@@ -15,6 +15,7 @@ export const styles = {
     fontSize: tokens.spacingM,
     fontFamily: tokens.fontStackPrimary,
     minHeight: '400px',
+    overflow: 'scroll',
```

Comment:
- I think this should be `overflow-y` as we don't want horizontal scrolling. 
- Any reason for `scroll` instead of `auto`?